packages/webpack-extraction-plugin/__fixtures__/webpack/basic-rules/output.ts

```diff
@@ -0,0 +1,16 @@
+import { __styles, __css } from '@griffel/react';
```

Comment:
```suggestion
import {  __css } from '@griffel/react';
```

Reply:
The same as https://github.com/microsoft/griffel/pull/135#discussion_r898997520, tests don't run in `production` mode so tree shaking is not happening.